tapdb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -315,7 +315,7 @@ CREATE TABLE IF NOT EXISTS asset_seedlings (
 CREATE VIEW genesis_info_view AS
     SELECT
         gen_asset_id, asset_id, asset_tag, assets_meta.meta_data_hash meta_hash,
-        output_index, asset_type, genesis_points.prev_out prev_out, 
+        output_index, asset_type, genesis_points.prev_out prev_out,
```

Comment:
I don't think we should modify an old migration file. Might make it more complicated to debug later.

Reply:
This PR only has a trailing space as diff. Are you suggesting we revert the contents of this migration to the ones present prior to #667 ? (remove `anchor_txid` from **this** PR?)